SummerSchool/RGSForumViewController.swift

```diff
@@ -8,14 +8,36 @@
 
 import UIKit
 
-class RGSForumViewController: RGSBaseViewController {
+class RGSForumViewController: RGSBaseViewController, UITableViewDelegate, UITableViewDataSource, UIScrollViewDelegate {
```

Comment:
i'm not exactly an expert in swift/iOS but does it make sense to have this class be both the view delegate and the data source?

Reply:
Delegate and data source methods are standard for using UITableViews. Being the DataSource simply means I have to implement methods that handle configuring the cells, and the delegate methods allow me to control table behavior.

Reply:
ok. you're the expert of this domain. though i should look into it more closely at some point :)